### ISSUE TITLE
The behavior of `ThrottleFirst`/`ThrottleFirstFrame` is not consistent with UniRx

### ISSUE THREAD

**Issue by aneuf-tech** [Cysharp/R3#49]
I found that the behavior of `ThrottleFirst`/`ThrottleFirstFrame` is not consistent with UniRx.

`ThrottleFirst`/`ThrottleFirstFrame` of R3 does not emit a value until `timeSpan`/`frameCount` has elapsed, even if it is the first value, while those of UniRx does not wait for `timeSpan`/`frameCount` when emitting the first value.

```cs
var startingTime = DateTime.Now;
Observable.Timer(TimeSpan.FromMilliseconds(100), TimeSpan.FromMilliseconds(100))
    .ThrottleFirst(TimeSpan.FromSeconds(2))
    .Subscribe(_ => Console.WriteLine((DateTime.Now - startingTime).TotalSeconds));

// 2.0087288
// 4.005542
// 6.0163164
// ...
```

The related test also assumes this behavior, and is this the expected?

This behavior makes it difficult to use for purpose that, for example, prevent double UI input...

**Comment by neuecc** [comment]
The point is correct, and it seems like a behavior that should be fixed!
I will check the behavior of other Rxs (RxJava, RxJS, etc.) including debounce, and then make the necessary corrections.

**Comment by aneuf-tech** [comment]
Thanks for your work!

**Comment by neuecc** [comment]
released `0.1.8`, thanks!
